scraper-pipeline/extract.py

```diff
@@ -7,20 +7,20 @@
 from bs4 import BeautifulSoup
 import requests
 
+# pylint: disable=too-few-public-methods
+
 
 class RSSFeedExtractor:
     '''The RSSFeed class extracts all articles on the inputted rss url,
       it also scrapes each individual article's body of content'''
 
     def __init__(self, rss_feeds: list[str]):
         self.rss_feeds = rss_feeds
-        self.extract = self.extract_feeds()
 
-    def body_extractor(self, link):
+    def _body_extractor(self, link):
```

Comment:
```suggestion
    def _body_extractor(self, link: str) -> str:
```
Double-check these type-hints are correct